internal/weldr/api.go

```diff
@@ -1207,23 +1193,12 @@ func (api *API) modulesInfoHandler(writer http.ResponseWriter, request *http.Req
 		statusResponseError(writer, http.StatusBadRequest, errors)
 		return
 	}
-	availablePackages, err := api.fetchPackageList(distroName)
-
-	if err != nil {
-		errors := responseError{
-			ID:  "ModulesError",
-			Msg: fmt.Sprintf("msg: %s", err.Error()),
-		}
-		statusResponseError(writer, http.StatusBadRequest, errors)
-		return
-	}
-
-	foundPackages, err := availablePackages.Search(names...)
+	foundPackages, err := api.fetchPackageList(distroName, names)
```

Comment:
Does this change the behaviour in any way compared to before?

Reply:
No, the results are actually the same, just faster.